ansible/roles/azure-cloud-storage/tasks/upload-using-azcopy.yml

```diff
@@ -1,12 +1,22 @@
 ---
+- name: generate SAS token for azcopy
+  shell: |
+   sas_expiry=`date -u -d "30 minutes" '+%Y-%m-%dT%H:%MZ'`
```

Comment:
I did not see this change in devops repo. Can we include this change in devops repo also?

Reply:
yes, will add there too.